src/app/workflow/register-workflow-modal/register-workflow-modal.component.html

```diff
@@ -92,42 +92,39 @@ <h4 class="modal-title">Register Workflow</h4>
                   <div *ngIf="formErrors.gitPath" class="alert alert-danger"> {{ formErrors.gitPath }} </div>
```

Comment:
This is beyond the scope of this PR, but just mentioning it since I was trying it out -- I find it slightly disconcerting that the page comes up with errors, before the user has even had a chance to input anything. The more common UX flow that I've seen is that the required errors are indicated after the user tries to submit. And then you could have some sort of indicator to show which fields are required.

Reply:
I agree with this

Reply:
I think ideally the required fields are marked with a star beforehand.  The validation errors start to appear once the specific field is touched.